src/env.js

```diff
@@ -5,11 +5,11 @@
   // turn on/off the console.log's
   window.__env.enableDebug = false;
 
-  // Environment name
+  //These will replaced by config.service.ts
   window.__env.ENVIRONMENT = 'local';  // local | dev | test | prod
 
   window.__env.API_LOCATION = 'http://localhost:3000';
-  window.__env.NRCED_LOCATION = 'http://localhost:8080';
+  window.__env.NRCED_LOCATION = 'https://nrced.gov.bc.ca/'
```

Comment:
what is the reason for using the prod NRCED url in the local env?

Reply:
This will be the default value if no /config from NRPTI is able to be retrieved, my thought process is that production would be the safest option in case this happens outside of a local environment, and will also still work locally.